lib/doorkeeper/openid_connect/response_types_config.rb

```diff
@@ -0,0 +1,17 @@
+module Doorkeeper
+  module OpenidConnect
+    module ResponseTypeConfig
+      private def calculate_authorization_response_types
+        types = super
+        if grant_flows.include? 'implicit_oidc'
```

Comment:
This probably requires some explanation in the README, I assume this needs to be enabled with `grant_flows` in `config/initializers/doorkeeper.rb`?